includes/blocks/class-sensei-course-outline-block.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * File containing the Sensei_Course_Outline_Block class.
+ *
+ * @package sensei
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class Sensei_Course_Outline_Block
+ */
+class Sensei_Course_Outline_Block {
+	/**
+	 * Sensei_Course_Outline_Block constructor.
+	 */
+	public function __construct() {
+		add_action( 'enqueue_block_assets', [ $this, 'enqueue_assets' ] );
```

Comment:
I explored the advantages between add the assets like this or in the `register_block_type`. I decided to go through this way because it more flexible - We can add multiple assets, add assets behind if statements (only for the frontend, for example), and so on.